Specs/DateTime/Dutch/HolidayParser.json

```diff
@@ -28,6 +28,7 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
+    "NotSupported": "dotnet",
```

Comment:
This one can be enabled again, I guess. Could you enable all testcases that are currently supported in the final PR?